src/components/Search/Search.tsx

```diff
@@ -5,6 +5,6 @@ import './Search.scss';
 export const Search: React.FC = () => (
   <div className="search">
     <SearchIcon className="search__icon" />
-    <input placeholder="Search..." className="search__input" type="text" />
+    <input data-testid="searchInput" placeholder="Search..." className="search__input" type="text" />
```

Comment:
it's not correct because you should pass testId as a prop (if we will have more than 1 search on the page then testId won't be unique)

Reply:
It refers to the whole PR! (for example dropdown)